cleanlab/multiannotator.py

```diff
@@ -275,26 +313,425 @@ def get_label_quality_multiannotator(
+        labels_multiannotator = pd.DataFrame(labels_multiannotator)
+
+    assert_valid_pred_probs(pred_probs=pred_probs, pred_probs_unlabeled=pred_probs_unlabeled)
+
+    num_classes = get_num_classes(pred_probs=pred_probs)
+
+    optimal_temp = find_best_temp_scaler(labels_multiannotator, pred_probs)
+    pred_probs = temp_scale_pred_probs(pred_probs, optimal_temp)
+
+    multiannotator_info = get_label_quality_multiannotator(
+        labels_multiannotator,
+        pred_probs,
+        return_annotator_stats=False,
+        return_detailed_quality=False,
+        return_weights=True,
+    )
+
+    quality_of_consensus_labeled = multiannotator_info["label_quality"]["consensus_quality_score"]
+    model_weight = multiannotator_info["model_weight"]
+    annotator_weight = multiannotator_info["annotator_weight"]
+    avg_annotator_weight = np.mean(annotator_weight)
+
+    # compute scores for labeled data
+    active_learning_scores = np.full(len(labels_multiannotator), np.nan)
+    for i in range(len(active_learning_scores)):
+        annotator_labels = labels_multiannotator.iloc[i]
+        active_learning_scores[i] = np.average(
+            (quality_of_consensus_labeled[i], 1 / num_classes),
+            weights=(
+                np.sum(annotator_weight[annotator_labels.notna()]) + model_weight,
+                avg_annotator_weight,
+            ),
+        )
+
+    # compute scores for unlabeled data
+    if pred_probs_unlabeled is not None:
+        pred_probs_unlabeled = temp_scale_pred_probs(pred_probs_unlabeled, optimal_temp)
+        quality_of_consensus_unlabeled = np.max(pred_probs_unlabeled, axis=1)
+
+        active_learning_scores_unlabeled = np.average(
+            np.stack(
+                [
+                    quality_of_consensus_unlabeled,
+                    np.full(len(quality_of_consensus_unlabeled), 1 / num_classes),
+                ]
+            ),
+            weights=[model_weight, avg_annotator_weight],
+            axis=0,
+        )
+
     else:
-        return {
-            "label_quality": label_quality,
-        }
+        active_learning_scores_unlabeled = np.array([])
+
+    return active_learning_scores, active_learning_scores_unlabeled
+
+
+def get_active_learning_scores_ensemble(
+    labels_multiannotator: Union[pd.DataFrame, np.ndarray],
+    pred_probs: np.ndarray,
+    pred_probs_unlabeled: Optional[np.ndarray] = None,
+) -> Tuple[np.ndarray, np.ndarray]:
+    """Returns an active learning qualityscore for each example in the dataset, which can be used to prioritize what data to collect additional labels for. We consider settings where one example can be labeled by multiple annotators and some examples have no labels at all so far. The score is in between 0 and 1; lower scores indicate examples whose true label we are less confident about based on the current data (the examples with lowest scores are those for which additional labels should be collected). For example:
```

Comment:
Let's reason by analogy here. The docstring for this function should be structured relative to the docstring for the `get_active_learning_scores()` function as the `get_multiannotator_label_quality_ensemble()` docstring currently is written relative to the  `get_multiannotator_label_quality()` function.

In both cases, the one-liner for the `_ensemble` version should already explain what is different from the non-ensemble version.  Then the second/third sentence of docstring should always say this method is similar to the non-ensemble version (with link to non-ensemble version) except it is for settings  where multiple models have been trained.

Reply:
For each of the args' individual description, you also want to link the user back to the non-ensemble version's docstring and keep the docstring in ensemble version more minimal